src/evaluate.jl

```diff
@@ -202,7 +198,7 @@ evaluate(a::HomogeneousPolynomial{T}, vals::Array{S,1} ) where
 
 evaluate(a::HomogeneousPolynomial, v, vals...) = evaluate(a, (v, vals...,))
 
-evaluate(a::HomogeneousPolynomial, v) = evaluate(a, v...)
+evaluate(a::HomogeneousPolynomial, v) = evaluate(a, [v...])
```

Comment:
Isn't this the same as just `evaluate(a, v)`?

Reply:
I think this is needed, because it was (at least partly) the cause of #242; see [this comment](https://github.com/JuliaDiff/TaylorSeries.jl/issues/242#issuecomment-632257884). So I'll keep it... test pass.